src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const columns = document.querySelectorAll('tr');
+
+columns.forEach((el) => {
+  const posColumn = el.cells[1];
```

Comment:
It's important to check if 'el.cells[1]' exists to avoid potential errors. If 'el' has fewer than two cells, this will cause an error.